tools/harness-automation/autothreadharness/harness_case.py

```diff
@@ -345,25 +361,32 @@ def setUp(self):
         if self.__class__ is HarnessCase:
             return
 
-        logger.info('Setting up')
+        logger.info("Setting up")
         # clear files
-        logger.info('Deleting all .pdf')
+        logger.info("Deleting all .pdf")
         os.system('del /q "%HOMEDRIVE%%HOMEPATH%\\Downloads\\NewPdf_*.pdf"')
```

Comment:
not sure but is there still have such NewPdf_*.pdf since R1.4.0?

Reply:
there is no pdf in Dowloads directory since R1.4.0. will update